staging/src/k8s.io/sample-apiserver/hack/update-codegen.sh

```diff
@@ -67,3 +67,5 @@ ${informergen} \
   --listers-package k8s.io/sample-apiserver/pkg/client/listers \
   --output-package k8s.io/sample-apiserver/pkg/client/informers
   "$@"
+
+cp -R $GOPATH/src/k8s.io/sample-apiserver/pkg/client $GOPATH/src/k8s.io/kubernetes/staging/src/k8s.io/sample-apiserver/pkg/  
```

Comment:
Does this mean that the upper clientgen creates the client at the wrong location?